examples/fine-tuned-RAG/ModelFinetune.ipynb

```diff
@@ -36,7 +36,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        vectors_config=VectorParams(size=384, distance=Distance.DOT),
</pre><p>We typically use cosine embedding — and that's what FastEmbed uses as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/openai-cookbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Changed. I copied from the documentation as is. Thanks for pointing.